lib/content-services/src/lib/search/components/search-filter-chips/search-facet-chip-tabbed/search-facet-chip-tabbed.component.ts

```diff
@@ -0,0 +1,173 @@
+/*!
+ * @license
+ * Copyright © 2005-2023 Hyland Software, Inc. and its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { Component, ElementRef, Inject, Input, OnChanges, OnInit, SimpleChanges, ViewChild, ViewEncapsulation } from '@angular/core';
+import { ConfigurableFocusTrap, ConfigurableFocusTrapFactory } from '@angular/cdk/a11y';
+import { MatMenuTrigger } from '@angular/material/menu';
+import { TabbedFacetField } from '../../../models/tabbed-facet-field.interface';
+import { Observable, Subject } from 'rxjs';
+import { SearchQueryBuilderService } from '../../../services/search-query-builder.service';
+import { SEARCH_QUERY_SERVICE_TOKEN } from '../../../search-query-service.token';
+import { FacetWidget } from '../../../models/facet-widget.interface';
+import { TranslationService } from '@alfresco/adf-core';
+import { SearchFacetFiltersService } from '../../../services/search-facet-filters.service';
+import { AutocompleteOption } from '../../../models/autocomplete-option.interface';
+
+@Component({
+  selector: 'adf-search-facet-chip-tabbed',
+  templateUrl: './search-facet-chip-tabbed.component.html',
+  styleUrls: ['./search-facet-chip-tabbed.component.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class SearchFacetChipTabbedComponent implements OnInit, OnChanges, FacetWidget {
+    @Input()
+    tabbedFacet: TabbedFacetField;
+
+    @ViewChild('menuContainer', { static: false })
+    menuContainer: ElementRef;
+
+    @ViewChild('menuTrigger', { static: false })
+    menuTrigger: MatMenuTrigger;
+
+    private resetSubject$ = new Subject<void>();
+
+    displayValue$: Subject<string> = new Subject<string>();
+    reset$: Observable<void> = this.resetSubject$.asObservable();
+    focusTrap: ConfigurableFocusTrap;
+    chipIcon = 'keyboard_arrow_down';
+    autocompleteOptions = {};
+    selectedOptions = {};
+
+    constructor(@Inject(SEARCH_QUERY_SERVICE_TOKEN) public queryBuilder: SearchQueryBuilderService,
```

Comment:
Do we need the queryBuilder to be public? Or can we set it as private. I don't see any usages of this components queryBuilder object outside of this class

Reply:
Good call, made it private